### DISCUSSION TITLE
Multiple accounts

### DISCUSSION THREAD

**Discussion by Ubergeeksov** [BottlecapDave/HomeAssistant-OctopusEnergy#363] [Q&A]
Hi,

It is possible to set up multiple Octopus accounts in this integration? They are all registered to me (so have the same API), but different account numbers (A-AXXXXXXX).

Thanks.

**Comment by BottlecapDave** [comment]
Hi there. There is an open issue for this https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/223. If I'm honest it's low down on my list as it would require a large rewrite of the integration (and complications in what's available in the current HA UI setup). I'm also trying to direct my time to features that would benefit wider portions of the community. 

Please vote on the issue so I can gauge interest.
